### PR TITLE
Fix wizard not remove widget

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

File: safe/gui/tools/wizard/step_kw45_inasafe_fields.py [hunk 1]
```diff
@@ -191,3 +191,11 @@ def get_inasafe_fields(self):
                 inasafe_fields[parameter.guid] = parameter.value
 
         return inasafe_fields
+
+    def clear(self):
```
Comment (Gustry) [inline safe/gui/tools/wizard/step_kw45_inasafe_fields.py]: Not possible to refactor this in the parent class ?

Comment (ismailsunni) [inline safe/gui/tools/wizard/step_kw45_inasafe_fields.py]: No, only 3 step from all steps that has similar behaviour.
We can, but the implementation is specific to each step. We can refactor it again later.

Comment (Gustry) [inline safe/gui/tools/wizard/step_kw45_inasafe_fields.py]: Ok, no worries

**Comment by Gustry** [comment]
lgtm

**Pull request merged**